core/cordova.js

```diff
@@ -947,7 +947,11 @@
                 action,
                 actionArgs
             };
-            window.webkit.messageHandlers.bridge.postMessage(command);
+            if (window.androidBridge) {
```

Comment:
I wonder if we could just export `window.webkit.messageHandlers` on Android as well? Then we would have to modify this? I haven't checked if that's possible yet though